spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/ribbon/DefaultServerIntrospector.java

```diff
@@ -30,8 +30,12 @@
  */
 public class DefaultServerIntrospector implements ServerIntrospector {
 
-	@Autowired
-	private ServerIntrospectorProperties serverIntrospectorProperties;
+	private ServerIntrospectorProperties serverIntrospectorProperties = new ServerIntrospectorProperties();
+
+	@Autowired(required = false)
```

Comment:
I used this because if I only use the auto configuration I have to change many unit tests to set this field via reflection. But if you want it I can change it?